iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergStorageHandler.java

```diff
@@ -732,6 +809,16 @@ private void addCustomSortExpr(Table table,  org.apache.hadoop.hive.ql.metadata.
       int order = fieldOrderMap.get(spec.getColumnName());
       if (TransformSpec.TransformType.BUCKET.equals(spec.getTransformType())) {
         customSortExprs.add(BUCKET_SORT_EXPR.apply(order + offset, spec.getTransformParam().get()));
+      } else if (TransformSpec.TransformType.TRUNCATE.equals(spec.getTransformType())) {
```

Comment:
that's enum, could we use switch instead?

Reply:
I have replaced the if-else block with switch block.